src/libespeak-ng/readclause.c

```diff
@@ -561,7 +562,11 @@ int ReadClause(Translator *tr, char *buf, short *charix, int *charix_top, int n_
 					c1 = GetC();
 				}
 				xml_buf2[n_xml_buf] = 0;
-				c2 = GetC();
+				if (Eof()) {
+					c2 = '\0';
+				} else {
+					c2 = GetC();
+				}
```

Comment:
Can you add a test case for these two changes to a new `tests/ssml` test file? That will be automatically picked up in the `tests/ssml.test` test case.